docs/data-management/preliminary.md

```diff
@@ -133,93 +228,4 @@ When a new session is added, your *DataLad* dataset will remain at the same poin
-
-        HPC systems typically recommend using their login nodes only for tasks related to job submission, data management, and preparing jobscripts.
-        Therefore, the execution of resource-intensive tasks such as *fMRIPrep* or building containers on login nodes can negatively impact the overall performance and responsiveness of the system for all users.
-        Interactive sessions are a great alternative when available and **should** be used when creating the *DataLad* dataset.
-        For example, in the case of systems operating SLURM, the following command would open a new interactive session:
-        ```
-        srun --nodes=1 --ntasks-per-node=1 --time=01:00:00 --pty bash -i
-        ```
-
-- [ ] Install *DataLad*.
-    Generally, the most convenient and user-sandboxed installation (i.e., without requiring elevated permissions) can be achieved by using *Conda*, but other alternatives (such as *lmod*) can be equally valid:
-
-    === "Install *DataLad* with *Conda*"
-
-        - [ ] Get and install *Conda* if it is not already deployed in the system:
-
-            ``` shell
-            wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-            bash Miniconda3-latest-Linux-x86_64.sh
-            ```
-
-        - [ ] Install *DataLad*:
-
-            ``` shell
-            conda install -c conda-forge -y "datalad>=0.19" datalad-container
-            ```
-
-    === "Install *DataLad* in HPC with *lmod* enabled"
-
-        - [ ] Check the availability and dependencies for a specific Python version (here we check 3.8.2):
-
-            ``` bash
-            module spider Python/3.8.2
-            ```
-
-        - [ ] Load Python (please note `ml` below is a shorthand for `module load`)
-
-            ``` bash
-            ml GCCcore/9.3.0 Python/3.8.2
-            ```
-
-        - [ ] Update *pip*:
-
-            ``` bash
-            python -m pip --user -U pip
-            ```
-
-        - [ ] Install *DataLad*:
-
-            ``` bash
-            python -m pip install --user "datalad>=0.19" datalad-container
-            ```
-
-- [ ] Check datalad is properly installed, for instance:
-
-    ``` shell
-    $ datalad --version
-    datalad 0.19.2
-    ```
-
-    ??? bug "*DataLad* crashes (*Conda* installations)"
-
-        *DataLad* may fail with the following error:
-        ``` py
-        ImportError: cannot import name 'getargspec' from 'inspect' (/home/users/cprovins/miniconda3/lib/python3.11/inspect.py)
-        ```
-
-        In such a scenario, create a *Conda* environment with a lower version of Python, and re-install datalad
-        ``` shell
-        conda create -n "datalad" python=3.10
-        conda activate datalad
-        conda install -c conda-forge datalad datalad-container
-        ```
-
-- [ ] Configure your Git identity settings.
-
-    ``` shell
-    cd ~
-    git config --global --add user.name "Jane Doe"
-    git config --global --add user.email doe@example.com
-    ```
-
```

Comment:
```suggestion
### HPC users - instructions to install *DataLad*
When HPC is planned for processing, *DataLad* will be required on that system(s).

- [ ] Start an interactive session on the HPC cluster

    ??? warning "Do not run the installation of *Conda* and *DataLad* in the login node"
    
        HPC systems typically recommend using their login nodes only for tasks related to job submission, data management, and preparing jobscripts.
        Therefore, the execution of resource-intensive tasks such as *fMRIPrep* or building containers on login nodes can negatively impact the overall performance and responsiveness of the system for all users.
        Interactive sessions are a great alternative when available and **should** be used when creating the *DataLad* dataset.
        For example, in the case of systems operating SLURM, the following command would open a new interactive session:
        ```
        srun --nodes=1 --ntasks-per-node=1 --time=01:00:00 --pty bash -i
        ```

- [ ] Install *DataLad*.
    Generally, the most convenient and user-sandboxed installation (i.e., without requiring elevated permissions) can be achieved by using *Conda*, but other alternatives (such as *lmod*) can be equally valid:

    === "Install *DataLad* with *Conda*"
        - [ ] Get and install *Conda* if it is not already deployed in the system:
            ``` shell
            wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
            bash Miniconda3-latest-Linux-x86_64.sh
            ```
        - [ ] Install *DataLad*:
            ``` shell
            conda install -c conda-forge -y "datalad>=0.19" datalad-container
            ```

    === "Install *DataLad* in HPC with *lmod* enabled"
        - [ ] Check the availability and dependencies for a specific Python version (here we check 3.8.2):
            ``` bash
            module spider Python/3.8.2
            ```
        - [ ] Load Python (please note `ml` below is a shorthand for `module load`)
            ``` bash
            ml GCCcore/9.3.0 Python/3.8.2
            ```
        - [ ] Update *pip*:
            ``` bash
            python -m pip --user -U pip
            ```
        - [ ] Install *DataLad*:
            ``` bash
            python -m pip install --user "datalad>=0.19" datalad-container
            ```

- [ ] Check datalad is properly installed, for instance:
    ``` shell
    $ datalad --version
    datalad 0.19.2
    ```

    ??? bug "*DataLad* crashes (*Conda* installations)"

        *DataLad* may fail with the following error:
        ``` py
        ImportError: cannot import name 'getargspec' from 'inspect' (/home/users/cprovins/miniconda3/lib/python3.11/inspect.py)
        ```
        In such a scenario, create a *Conda* environment with a lower version of Python, and re-install datalad
        ``` shell
        conda create -n "datalad" python=3.10
        conda activate datalad
        conda install -c conda-forge datalad datalad-container
        ```

- [ ] Configure your Git identity settings.
    ``` shell
    cd ~
    git config --global --add user.name "Jane Doe"
    git config --global --add user.email doe@example.com
    ```

```

Reply:
(I'd prefer you undo the changes on your branch rather than accepting this comment btw)